ui/component/publish/livestream/publishLivestream/view.jsx

```diff
@@ -0,0 +1,300 @@
+// @flow
+// import { SITE_NAME, WEB_PUBLISH_SIZE_LIMIT_GB } from 'config';
+// import type { Node } from 'react';
+// import * as ICONS from 'constants/icons';
+import React, { useState, useEffect } from 'react';
+// import { regexInvalidURI } from 'util/lbryURI';
+// import PostEditor from 'component/postEditor';
+// import FileSelector from 'component/common/file-selector';
+// import Button from 'component/button';
+import Card from 'component/common/card';
+import { FormField } from 'component/common/form';
+import Spinner from 'component/spinner';
+// import I18nMessage from 'component/i18nMessage';
+// import usePersistedState from 'effects/use-persisted-state';
+import * as PUBLISH_MODES from 'constants/publish_types';
+import PublishName from '../../shared/publishName';
+import CopyableText from 'component/copyableText';
+import Empty from 'component/common/empty';
+import moment from 'moment';
+import classnames from 'classnames';
+import ReactPaginate from 'react-paginate';
+import { SOURCE_NONE, SOURCE_SELECT, SOURCE_UPLOAD } from 'constants/publish_sources';
+
+type Props = {
+  uri: ?string,
+  mode: ?string,
+  // name: ?string,
+  title: ?string,
+  filePath: string | WebFile,
+  // fileMimeType: ?string,
+  isStillEditing: boolean,
+  balance: number,
+  doUpdatePublishForm: ({}) => void,
+  disabled: boolean,
+  publishing: boolean,
+  // doToast: ({ message: string, isError?: boolean }) => void,
+  inProgress: boolean,
+  // doClearPublish: () => void,
+  // ffmpegStatus: any,
+  optimize: boolean,
+  // size: number,
+  // duration: number,
+  // isVid: boolean,
+  // subtitle: string,
+  // setPublishMode: (string) => void,
+  // setPrevFileText: (string) => void,
+  // header: Node,
+  livestreamData: LivestreamReplayData,
+  // isLivestreamClaim: boolean,
+  checkLivestreams: (string, string) => void,
+  channelName: string,
+  channelId: string,
+  isCheckingLivestreams: boolean,
+  setWaitForFile: (boolean) => void,
+  setOverMaxBitrate: (boolean) => void,
```

Comment:
`setOverMaxBitrate` is not checked here.  Not familiar enough to know if it's irrelevant to livestream, or something that isn't handled here.

If irreleveant, the parent can also remove the state.

Reply:
Removed this and a few other upload related states like size, duration etc. in this file.